internal/clients/team3/clientforaging.go

```diff
@@ -73,13 +69,20 @@ func (c *client) DecideForage() (shared.ForageDecision, error) {
 		decay = (float64(sumOfCaught) + 1) * float64(numberOfHunters) * 0.5
 	}
 
-	coef = math.Max(c.params.riskFactor-0.1*decay, 0.1)
+	// A bit arbitrary
+	coef = math.Max(c.params.riskFactor-0.01*(1-c.params.riskFactor)/4*decay, 0.1)
```

Comment:
decay can be 0

Reply:
https://play.golang.org/p/53O2kIyIQl-

Reply:
Hold up, this isn't a div by 0 - Victor 2021

Reply:
decay can be 0 indeed https://play.golang.org/p/JcEQbbhBTjU. 
Here is a quick maths problem for you ;)
![merlin_158743359_ff291f8a-d473-4849-9d81-9762826b55f4-superJumbo](https://user-images.githubusercontent.com/34316180/104532915-a0f86c00-5611-11eb-87b9-353f974ca14a.jpg)

Reply:
![image](https://user-images.githubusercontent.com/33488131/104730299-afd64000-5731-11eb-8b98-db5c40aff9a7.png)

Reply:
too many params man soz my eyes failed me